src/components/ContactCard/ContactIdentity.jsx

```diff
@@ -2,20 +2,11 @@ import React from "react";
 import PropTypes from "prop-types";
 import { Avatar } from "cozy-ui/react/Avatar";
 import contactPropTypes from "../ContactPropTypes";
-
-const getFullContactName = name =>
-  ["namePrefix", "givenName", "additionalName", "familyName", "nameSuffix"]
-    .map(part => name[part])
-    .join(" ")
-    .trim();
+import { getFullContactName, getInitials } from "../../helpers/contacts";
```

Comment:
I'm not at ease with these helper functions in another file, I think we're losing some kind of consistency. What do you think of using the `ViewModel` pattern here (sometimes called `Presenter` pattern in the Rails world), that is: having a class that wrap a raw `io.cozy.contacts` object and that handles view logic like sorting the fields, generating the initials, etc.
`ContactCard` would become something like this: 
```jsx
const ContactCard = ({ title, contact, renderActions }) => {
  const viewModel = new ContactViewModel(contact)
  return (
    <div>
      <header className="contact-card-header">
        <ContactIdentity initials={viewModel.getInitials()} fullname={viewModel.fullname()} groups={[]} />
        {renderActions()}
      </header>
      <ContactFields fields=viewModel.getOrderedFields()} title={title} />
    </div>
  );
};
```

Reply:
Well, i don't know, what would be the advantages of using such a `ViewModel` vs pure, standalone functions ? I have nothing against it per se, but I don't see why it would be compelling either.